ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Tests.scala

```diff
@@ -11,6 +11,7 @@ object Tests {
 
   val default: Tests = Map(
     "SemanticTests" -> (new SemanticTests(_)),
+    "ClosedWorldIT" -> (new ClosedWorld(_)),
```

Comment:
If you have to add a test and exclude it everywhere it's probably a good indication that it should _not_ go in the tests that are run by default. Please move it to the `optional` tests underneath.

Reply:
We actually want to run it as default going forward (i.e. when the sandbox will move to kvutils and will pre-allocate parties) and it's excluded in the sandbox and Canton alone for now; @meiersi-da what do you think?

Reply:
You can use `--all-tests`, which is what you already do I believe. If we want to change the default behavior of the application I'm absolutely agree with it, but it should be a concern for a different PR.

Reply:
Yes, the separation of concerns argument is one I can agree to, even though if that's the agreed direction I'd start moving there sooner rather than later (but this can also be achieved with a separate PR). Leaving this pending for @meiersi-da to express his view but I agree to the change.

Reply:
Agreed offline to apply this change to all tests, removing blocker.

Reply:
@fabiotudone-da thanks for asking -- I'd suggest to follow @stefanobaghino-da 's advice.

Reply:
Feel free to keep this in at this point, in a few minutes #4561 will hopefully be merged and you can add this test to the default. Thanks for the feedback.